test/extended/networking.sh

```diff
@@ -7,9 +7,19 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-if [[ "${OPENSHIFT_QUIET_OUTPUT:-false}" != "true" ]]; then
+CONFIG_ROOT="${OPENSHIFT_CONFIG_ROOT:-}"
+
+# CI mode (deploying clusters for each plugin) should be noisy by
+# default to aid in debugging.
+if [[ "${CONFIG_ROOT}" = "" ]]; then
+  DEFAULT_QUIET=false
+else
+  DEFAULT_QUIET=true
+fi
+
+if [[ "${OPENSHIFT_QUIET_OUTPUT:-${DEFAULT_QUIET}}" != "true" ]]; then
```

Comment:
On Thu, Apr 14, 2016 at 8:38 PM, Steve Kuznetsov <notifications@github.com>
wrote:
>
> Simply because in Bash, the presence of an environment variable is
> equivalent to a boolean true -- if I set $OPENSHIFT_QUIET_OUTPUT, I want
> the output to be quiet.
>

My confusion wasn't around quiet being set = true, unset = false.  I didn't
understand why the logic you proposed was using `-n $OPENSHIFT_QUIET_OUTPUT`
as one indication that verbose output should be configured.  Did you mean
to use `-z`?